resources/css/filament/admin/theme.css

```
@@ -1,4 +1,220 @@
 @import "../../../../vendor/filament/filament/resources/css/theme.css";
 
+/* Google Fonts injected via Filament render hook (AppServiceProvider::boot).
+ * Family names mirror the docs (custom.css lines 24-47):
+ *   - Geist        (body sans-serif)
+ *   - Encode Sans  (display/headings)
+ *   - Geist Mono   (monospace) */
+
 @source '../../../../app/Filament/**/*';
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)